packages/python-google-compute-engine/google_compute_engine/instance_setup/instance_setup.py

```diff
@@ -28,9 +28,21 @@
 from google_compute_engine import logger
 from google_compute_engine import metadata_watcher
 from google_compute_engine.boto import boto_config
+from google_compute_engine.compat import urlerror
+from google_compute_engine.compat import urlrequest
 from google_compute_engine.instance_setup import instance_config
 
 
+class PutRequest(urlrequest.Request):
+    def get_method(self):
```

Comment:
The new indentation is wrong. The code should conform to Google Python style with 2 indents (4 for line continuations).

Reply:
Fixed. (I did some development on VMs and vi on those VMs defaults to 4 space indents. Some of it came over when I copied from the VMs.)